tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -57,11 +57,11 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
     {
         if (isRunningInAas)
         {
-            Logger::Info("The Tracer Profiler is initialized multiple times. This is expected and currently unavoidable when running in AAS.");
+            Logger::Info("Instrumentation is initialized multiple times. This is expected and currently unavoidable when running in AAS.");
         }
         else
         {
-            Logger::Error("The Tracer Profiler is initialized multiple times. This may cause unpredictable failures.",
+            Logger::Error("Instrumentation is initialized multiple times. This may cause unpredictable failures.",
                 " When running aspnetcore in IIS, make sure to disable managed code in the application pool settings.",
                 " https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/advanced?view=aspnetcore-9.0#create-the-iis-site");
```

Comment:
If you remove the aspnetcore version, the page will automatically selected the latest version, so we don't need to update this when new versions are released.
```suggestion
                " https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/advanced#create-the-iis-site");
```